### PR TITLE
feat(console): add hover tooltip for micro user avatars

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Add user info hover tooltip for audit log details and role assignment user selection panel.

## Testing

image

image

## Checklist

- [x] This PR is not applicable for the checklist

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]
"nano user avatars"

**Review by simeng-li — COMMENTED** [review]
typo in the PR body

File: packages/console/src/components/RoleUsersTransfer/components/SourceUserItem/index.tsx [hunk 1]
```diff
@@ -36,8 +33,8 @@ function SourceUserItem({ user, isSelected, onSelect }: Props) {
           onSelect();
         }}
       />
-      <UserAvatar user={user} size="micro" />
-      <div className={styles.title}>{getUserTitle(user) ?? t('users.unnamed')}</div>
+      <UserAvatar hasTooltip user={user} size="micro" />
```
Comment (simeng-li) [inline packages/console/src/components/RoleUsersTransfer/components/SourceUserItem/index.tsx:36]: nit: do you think implement a ToolTip wrapper could be a better approach?  

```jsx
<WithUserTip>
  <UserAvatar />
</WithUserTip>
```

Comment (charIeszhao) [reply packages/console/src/components/RoleUsersTransfer/components/SourceUserItem/index.tsx:36]: That's also doable, but personally I don't have a preference on which one is better than the other one.

Comment (charIeszhao) [reply packages/console/src/components/RoleUsersTransfer/components/SourceUserItem/index.tsx:36]: I've been thinking about this and it's actually a bit cumbersome than it sounds like. Also there's no other component can use this `<WithUserTip />` component atm. So I decided to save the effort and go ahead without this change.

**Pull request merged**